src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -740,7 +740,7 @@ To create an Azure infrastructure, ProActive relies on an existing Azure user ac
 * Access a https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups[resource group] of your choice.
 * Create VMs in that resource group.
 * Delete VMs in that resource group.
-- *Service Principal credentials:* ProActive uses Azure's API to manage the VMs in the infrastructure. This API requires a set of Active Directory _service principal credentials_ for non interactive login (client, secret, tenant). If you do not have them, you can create them via the https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal[Azure Portal] or https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest[Azure CLI]. Please ask your administrator in case you do not have the privileges to create them. The credentials should be strings of hexadecimal values like _1a2b3c4d-5e6f-1a2b-3c4d-5e6f-1a2b3c4d5e6f_.
+- *Service Principal credentials:* ProActive uses Azure's API to manage the VMs in the infrastructure. This API requires a set of Active Directory _service principal credentials_ for non interactive login (client, secret, tenant). If you do not have them, you can create them via the https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal[Azure Portal] or https://learn.microsoft.com/en-us/cli/azure/azure-cli-sp-tutorial-1[Azure CLI]. Please ask your administrator in case you do not have the privileges to create them. The credentials should be strings of hexadecimal values like _1a2b3c4d-5e6f-1a2b-3c4d-5e6f-1a2b3c4d5e6f_.
```

Comment:
Azure's API is a generic term (since Azure exposes a lot of APIs). I suggest to be more precise about the name if possible (or provide a link if possible).